.env.example

```
@@ -32,6 +32,16 @@ DISABLE_KEY_ID=''
 # NAME_LABEL is an optional username label for name registration (without .dash)
 NAME_LABEL=''
 
+# Optional second funded identity used by document purchase and transfer tests
+SECONDARY_PLATFORM_MNEMONIC='your mnemonic phrase goes here ...'
+
+# DOCUMENT_RECIPIENT_ID is the new owner identity ID for document-transfer.mjs
+DOCUMENT_RECIPIENT_ID=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)